doc/source/advanced.rst

```diff
@@ -113,7 +113,14 @@ of the index is up to you:
    pd.DataFrame(np.random.randn(6, 6), index=index[:6], columns=index[:6])
 
 We've "sparsified" the higher levels of the indexes to make the console output a
-bit easier on the eyes.
+bit easier on the eyes. Note that how the index is displayed can be controlled using the
+``multi_sparse`` option in ``pandas.set_options()``:
+
+.. ipython:: python
+
+   pd.set_option('display.multi_sparse', False)
```

Comment:
use these with the context manager, e.g. ``pd.option_context('display.multi_sparse', False)``

Reply:
Done